pkg/framework/aries/framework.go

```diff
@@ -128,6 +130,21 @@ func WithInboundTransport(inboundTransport transport.InboundTransport) Option {
 	}
 }
 
+// WithTransportReturnRoute injects transport return route option to the Aries framework. Acceptable values - "none",
```

Comment:
Do we need a TODO for more dynamically setting the TransportReturnRoute (in the protocol clients) or do we expect to always use the same setting (for a particular framework?

Reply:
We can enhance this to support route option at message/protocol layer by passing a `{messageType, returnValue}` config. eg, {`didexchange, all},{introduce, none}`. Created an issue - https://github.com/hyperledger/aries-framework-go/issues/899